resources/azure/compute/setup-ssh.ps1

```diff
@@ -0,0 +1,16 @@
+param(
+    [Parameter(Mandatory,HelpMessage='authorizedkey for ssh private key for the user')]
+    $authorizedKey
+)
+
+$service = Get-Service -Name sshd -ErrorAction SilentlyContinue
+if ($service -ne $null) {
+  Stop-Service sshd
+}
+Add-WindowsCapability -Online -Name OpenSSH.Server~~~~0.0.1.0
```

Comment:
Note for reviewers: this single line takes around one minute, an alternative is using a separate extension in `vm.go` and having the `NewVirtualMachineRunCommandByVirtualMachine ` depending on it. Content should be

```go
sshSetupExtension, err := compute.NewVirtualMachineExtension(e.Ctx(), "enable-ssh", &compute.VirtualMachineExtensionArgs{
		VmName:                  vm.Name,
		ResourceGroupName:       pulumi.String(e.DefaultResourceGroup()),
		Publisher:               pulumi.String("Microsoft.Azure.OpenSSH"),
		Type:                    pulumi.String("WindowsOpenSSH"),
		TypeHandlerVersion:      pulumi.String("3.0"),
		VmExtensionName:         pulumi.String("WindowsOpenSSH"),
		AutoUpgradeMinorVersion: pulumi.Bool(true),
	}, e.WithProviders(config.ProviderAzure), pulumi.Parent(vm))
```

Reply:
Never mind, I tried it and it took even longer to setup

```
+   pulumi:pulumi:Stack                                                  dd-paola-ducolin-pippo                       created (267s)      
 +   ├─ pulumi:providers:random                                           random                                       created (0.00s)     
 +   ├─ dd:Host                                                           az-vm                                        created (264s)      
 +   │  └─ command:remote:Command                                         remote-az-vm-cmd-wait-successful-connection  created (3s)        
 +   ├─ pulumi:providers:command                                          command                                      created (0.00s)     
 +   ├─ pulumi:providers:azure-native                                     azure                                        created (0.00s)     
 +   ├─ random:index:RandomString                                         az-az-vm-admin-password                      created (0.00s)     
 +   ├─ azure-native:network:NetworkInterface                             az-az-vm                                     created (5s)        
 +   └─ azure-native:compute:VirtualMachine                               az-az-vm                                     created (66s)       
 +      ├─ azure-native:compute:VirtualMachineExtension                   enable-ssh                                   created (125s)      
 +      └─ azure-native:compute:VirtualMachineRunCommandByVirtualMachine  az-vm-init-cmd                               created (64s) 
```